packages/transformers/js/core/src/dependency_collector.rs

```diff
@@ -49,6 +49,16 @@ pub enum DependencyKind {
   /// import('./dependency').then(({x}) => {/* ... */});
   /// ```
   DynamicImport,
+  /// Corresponds to a for display (tier) import statement
+  /// ```skip
+  /// const {x} = importDeferredForDisplay('./dependency');
```

Comment:
```suggestion
  /// const DependencyDeferred = importDeferredForDisplay('./dependency');
```
Can't destructure this, it's a class instance.